actors/power/tests/power_actor_tests.rs

```diff
@@ -58,3 +69,138 @@ fn create_miner() {
     verify_empty_map(&rt, st.cron_event_queue);
     h.check_state();
 }
+
+#[test]
+fn create_miner_given_caller_is_not_of_signable_type_should_fail() {
+    let (h, mut rt) = setup();
+
+    let peer = "miner".as_bytes().to_vec();
+    let multiaddrs = vec![BytesDe("multiaddr".as_bytes().to_vec())];
+
+    let create_miner_params = CreateMinerParams {
+        owner: *OWNER,
+        worker: *OWNER,
+        window_post_proof_type: RegisteredPoStProof::StackedDRGWindow32GiBV1,
+        peer,
+        multiaddrs,
+    };
+
+    rt.set_caller(*MINER_ACTOR_CODE_ID, *OWNER);
+    rt.expect_validate_caller_type(CALLER_TYPES_SIGNABLE.to_vec());
+    expect_abort(
+        ExitCode::USR_FORBIDDEN,
+        rt.call::<PowerActor>(
+            Method::CreateMiner as u64,
+            &RawBytes::serialize(&create_miner_params).unwrap(),
+        ),
+    );
+    rt.verify();
+    h.check_state();
+}
+
+#[test]
+fn create_miner_given_send_to_init_actor_fails_should_fail() {
+    let (h, mut rt) = setup();
+
+    let peer = "miner".as_bytes().to_vec();
+    let multiaddrs = vec![BytesDe("multiaddr".as_bytes().to_vec())];
+
+    let create_miner_params = CreateMinerParams {
+        owner: *OWNER,
+        worker: *OWNER,
+        window_post_proof_type: RegisteredPoStProof::StackedDRGWindow32GiBV1,
+        peer: peer.clone(),
+        multiaddrs: multiaddrs.clone(),
+    };
+
+    // owner send CreateMiner to Actor
+    rt.set_caller(*ACCOUNT_ACTOR_CODE_ID, *OWNER);
+    rt.value_received = TokenAmount::from(10);
+    rt.set_balance(TokenAmount::from(10));
+    rt.expect_validate_caller_type(vec![*ACCOUNT_ACTOR_CODE_ID, *MULTISIG_ACTOR_CODE_ID]);
```

Comment:
nit: you can improve on the go test by using `CALLER_TYPES_SIGNABLE.to_vec()` instead as you did above